### DISCUSSION TITLE
GitHub Mobile support for Enterprise Cloud

### DISCUSSION THREAD

**Discussion by elyscape** [community/community#30957] [Enterprise]
The iPhone GitHub app supports simultaneous login to personal and GitHub Enterprise accounts. Unfortunately, it appears that it only supports GitHub Enterprise Server. It would be nice if it also supported GitHub Enterprise Cloud.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.77.0 (76215101)
Device: iPhone 13 Pro (iOS 15.6.1)
</details>

**Comment by alialhammad1992** [comment]
We hope Github supports iCloud

**Comment by Bazfred** [comment]
Need a gift for mag box

**Comment by candyho** [comment]
Hey thanks for your feedback. Supporting multi-account is something already on our backlog. I will share this feedback with the team and hopefully we'll have some updates to share in the near future.

**Reply by leonmax** [reply]
Hi there, it seems like we are discussing the issue of logging into "Github Enterprise Cloud" on Github Mobile, rather than multi-account usage. As paying customers, we would appreciate some attention from the mobile team regarding this matter. Thank you

**Reply by candyho** [reply]
Hey @leonmax you should already be able to login to GitHub Mobile with your Enterprise Cloud account. Since GitHub Mobile doesn't support multi-account switching yet, if you're already signed in with your personal account, you would need to first sign out of the app, then login again with your Enterprise Cloud by choosing `Sign in with Github` on the login screen - hope this helps!

**Comment by thehamsti** [comment]
Hi not to revive a dead thread but was wondering if there was any update on the status of this? @candyho

**Reply by candyho** [reply]
We are actively working on multi-account support. Hopefully there's some news to share in the near future. Stay tuned!
